scriptmodules/supplementary/usbromservice/20_retropie_overlayfs.mount

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
these scripts should use the default shell and not rely on bash.

Reply:
Sorry - ignore this. It's ok to use bash (and we use it already - although possibly we shouldn't for this).